mmdet3d/core/visualizer/image_vis.py

```diff
@@ -56,46 +56,31 @@ def project_pts_on_img(points,
     cv2.waitKey(100)
 
 
-def project_bbox3d_on_img(bboxes3d,
```

Comment:
I remove this function as it is not used in the codebase. Its name is very ambiguous since we already have `project_camera/lidar/depth_bbox` now. Also it works almost the same as `draw_lidar_bbox3d_on_img`.